docs/notes/platforms.md

```diff
@@ -28,4 +28,24 @@ The Mac engine supports:
 *10.7.x (Lion) on Intel
 *10.8.x (Mountain Lion) on Intel
 *10.9.x (Mavericks) on Intel
+*10.10.x (Yosemite) on Intel
+*10.11.x (El Capitan) on Intel
 * **Note:** The engine runs as a 32-bit application regardless of the capabilities of the underlying processor.*
+## iOS
+iOS deployment is possible when running LiveCode IDE on a Mac, and provided Xcode is installed and has been set in LiveCode *Preferences* (in the *Mobile Support* pane).
+Currently, the supported versions of Xcode are:
+*Xcode 4.6 on MacOS X 10.6
```

Comment:
Hm good point, in `revdeploylibraryios.livecodescript`, we simply take Xcode 4.6 for MacOS X < 10.8, but I didn't think that it was not the same for 10.7 and 10.6.
I'll remove the line for 10.6